index.js

```diff
@@ -58,6 +61,25 @@ function handleRTCMessage(client, payload)
   }
 }
 
+// Make sure any updates to this are reflected in README
+const statErrorNames = [
+  "Abort",
+  "Hardware",
+  "NotFound",
+  "NotSupported",
+  "Permission",
+  "SecureConnection",
+  "Unknown"
+]
+
+function handleErrorStatMessage(statName) {
+  if (statErrorNames.indexOf(statName) !== -1) {
+    stats.meter("ep_webrtc_err_" + statName).mark();
+  } else {
+    statsLogger.warn("Invalid ep_webrtc error stat: " + statName);
```

Comment:
I decided to make a log entry instead, to follow how I saw invalid messages were handled.